lib/harvesting/models/project_task_assignments.rb

```diff
@@ -0,0 +1,39 @@
+module Harvesting
+  module Models
+    class ProjectTaskAssignments < Base
```

Comment:
@mscottford I believe you can now extend from `HarvestRecordCollection` and DRY this class.

Reply:
Done.